infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -106,14 +118,13 @@ public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
         // Store clientID as a channel attribute.
         ctx.channel().attr(clientIdAttrKey).set(clientId);
         log.info("channelRead: Handshake validated by Server.");
-        log.debug("channelRead: Sending handshake response: Node Id: {} Corfu Version: {}",
-                this.nodeId, this.corfuVersion);
+        log.debug("channelRead: Sending handshake response: Node Id: {}", this.nodeId);
```

Comment:
Can we print a debug message of the server version like before? Might help in bug triaging as this is the only place where we are doing main version checks.

```
log.debug("channelRead: Sending handshake response: Node Id: {}, ServerVersion: {}, ClientVersion: {}", this.nodeId, serverVersion.toString(), clientVersion.toString());
````

Reply:
Added, thanks!